core/tests/widgets/test_container_debug_mode.py

```diff
@@ -0,0 +1,76 @@
+from pytest import MonkeyPatch
+from travertino.colors import color
+
+import toga
+
+
+# test that a container-like widget in normal mode has a default background
+def test_box_no_background_in_normal_mode():
+    """A Box has no default background."""
+    # Disable layout debug mode
+    with MonkeyPatch.context() as mp:
+        mp.setenv("TOGA_DEBUG_LAYOUT", "0")
```

Comment:
Monkeypatch can be used as a fixture:
```suggestion
def test_box_no_background_in_normal_mode(monkeypatch):
    """A Box has no default background."""
    # Disable layout debug mode
    monkeypatch.setenv("TOGA_DEBUG_LAYOUT", "0")
```